src/routes/landing/components/Landing.js

```diff
@@ -41,6 +41,7 @@ class Landing extends React.Component<PropType> {
 
   render () {
     const {theme, cities, t} = this.props
+    // $FlowFixMe Going to fix this in this task: react-native uses now strict types
```

Comment:
Going to fix these in a separate task.

Reply:
Did you already create a task?

Reply:
Yes: https://issues.integreat-app.de/browse/NATIVE-292

With the new flow types this $FlowFixMe is not needed anymore because the flow types are a bit buggy..